### ISSUE TITLE
Compare the OneHotAutoEncocder and SImpleConvAutoEncoder wiht a traditional approach using input shape of (32,24,24,4) e.g RGBT

### ISSUE THREAD

**Issue by tnn1t1s** [jmsdnns/pallets#4]
add a new conv model that does not operate on one hot encocded inputs but instead uses conventional approach of predicting on RGB. . in the notebook, compare this apporach with the onehotencoded approach.

**Comment by jmsdnns** [comment]
The underlying theory of the pallets project is that a one hot encoding scheme can be used to create color palettes for image generation, where a single shade of brown would be used everywhere brown appears. In theory, this would be an improvement over a more typical RGBA model where brown might be shown in multiple variations that are close to each other, but visibly inconsistent.

Using the models commited [here](https://github.com/tnn1t1s/pallets/blob/444bb8be5859d19d84db5577ae9acb3d0607ba7c/pallets/models.py) we show that this is indeed true.

The one hot encoded model uses a cross entropy loss function and the RGBA model uses MSE, so comparing scores is not helpful. However, just looking at the images reveals what we were looking for.

## Naive RGBA

It is plain to see the many shades of brown that emerge with a 4 channel RGBA model. In this case, the model is very basic, giving us an image with all kinds of color problems in it. (_what we wanted_)

![naive-rgba](https://github.com/tnn1t1s/pallets/assets/7950614/ee04c98d-7305-459a-b0cb-ef0cc7f1e40b)

## Naive One Hot

We can see how consistently the color palette is applied by using the same naive model, except for special handling to cover the one hot encoded colors.

![naive-onehot](https://github.com/tnn1t1s/pallets/assets/7950614/0a1f5084-cc5f-46dc-b337-c694d7c5e3b8)

## Conv2d RGBA

This is another 4 channel model, but based on a more sophisticated conv2d net. The colors are much better than the naive model, but if we look closely we can still variations amongst the colors. Look around the eyes and strands of hair on top of the head for several colors resembling beige.

![conv-rgba](https://github.com/tnn1t1s/pallets/assets/7950614/76015db4-583c-413f-8cf0-2ac0ad518f99)

## Conv2d One Hot

This is the best quality model so far. The color palette is applied very well to an image with well defined features, such as eye makeup, cigarette, and hair/skin color.

![conv-onehot](https://github.com/tnn1t1s/pallets/assets/7950614/12baa3b3-51f9-4ab5-a70e-260be88b9be6)

**Comment by jmsdnns** [comment]
I believe this can be closed, but would like @tnn1t1s to make that call.

**Comment by jmsdnns** [comment]
this is indeed done
